bin/morgue.js

```diff
@@ -7437,6 +7438,136 @@ async function metricsImporterCmd(argv, config) {
   await cli.routeMethod(argv);
 }
 
+function stabilityCreateMetric(coroner, argv, config) {
+  let universe = argv.universe;
+  if (!universe) {
+    universe = Object.keys(config.config.universes)[0];
+  }
+
+  let project = argv.project;
+
+  if (!universe) {
+    errx("--universe is required");
+  }
+
+  if (!project) {
+    errx("--project is required");
+  }
+
+  let bpg = coronerBpgSetup(coroner, argv);
+  let model = bpg.get();
+
+  /* Find universe. */
+  let uid = 0;
+  for (let u of model.universe) {
+    if (u.get("name") === universe) {
+      uid = u.get("id");
+    }
+  }
+
+  if (uid == 0) {
```

Comment:
We should keep using javascript good practices. In this situation we should use `===` instead of `==`. In case when something wrong happened and we will return somehow `false` or `undefined` or `null` then this condition will be false and we will continue code execution. I think we should: 
1. use `===` 
2. or !uid - negation will handle 0 as well.